### PR TITLE
feat(mesure-service): handle service_id in mesures

### PR DESCRIPTION
**Pull request by tglatt**

### CODE REVIEW

File: packages/api/auth/local.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ passport.use(
         .where("username", username)
         .orWhere("email", username.toLowerCase().trim())
         .first()
-        .eager("[roles, antennes, tis]")
+        .eager("[roles, service, antennes, tis]")
```
[the remark this answers is not in the document]
Comment (tglatt) [reply packages/api/auth/local.js]: I don't think so, but I did not dare :-)

File: packages/app/src/components-v2/ServiceMesures/ServiceAcceptMesure.js [hunk 2]
```diff
@@ -37,14 +37,23 @@ export const ServiceAcceptMesure = props => {
...
+                antenne_id: values.antenne_id ? values.antenne_id.value : null,
                 code_postal: values.code_postal,
                 date_ouverture: values.date_ouverture,
                 id: currentMesure,
                 residence: values.residence.value,
                 ville: values.ville
               }
             });
+            if (values.antenne_id) {
+              UpdateAntenneCounters({
```
Comment (kwiss) [inline packages/app/src/components-v2/ServiceMesures/ServiceAcceptMesure.js]: move me in first mutation success

File: packages/app/src/components-v2/ServiceMesures/ServiceAddMesure.js [hunk 3]
```diff
@@ -88,18 +90,28 @@ export const ServiceAddMesure = props => {
...
                     numero_rg: values.numero_rg,
                     residence: values.residence.value,
+                    service_id: serviceAdmin.service_id,
                     ti_id: values.tribunal.value,
                     type: values.type.value,
                     ville: values.ville
                   }
                 });
+                if (values.antenne_id) {
+                  UpdateAntenneCounters({
```
Comment (kwiss) [inline packages/app/src/components-v2/ServiceMesures/ServiceAddMesure.js]: move me in first mutation success

File: packages/app/src/components-v2/ServiceMesures/ServiceCloseMesure.js [hunk 4]
```diff
@@ -56,12 +57,21 @@ export const ServiceCloseMesure = props => {
...
-                  antenne_id: mesure.antenne_id,
                   extinction: values.extinction,
                   id: currentMesure,
-                  reason_extinction: values.reason_extinction.value
+                  reason_extinction: values.reason_extinction.value,
+                  service_id: mesure.service_id
                 }
               });
+              if (mesure.antenne_id) {
+                UpdateAntenneCounters({
```
Comment (kwiss) [inline packages/app/src/components-v2/ServiceMesures/ServiceCloseMesure.js]: move me in first mutation success

File: packages/app/src/components-v2/ServiceMesures/ServiceEditMesure.js [hunk 5]
```diff
@@ -86,19 +75,37 @@ export const ServiceEditMesure = props => {
...
+              UpdateAntenneCounters({
+                variables: {
+                  antenne_id: values.antenne_id.value,
+                  inc_mesures_awaiting: 0,
+                  inc_mesures_in_progress: 1
+                }
+              });
+            }
+            if (antenneId) {
+              UpdateAntenneCounters({
```
Comment (kwiss) [inline packages/app/src/components-v2/ServiceMesures/ServiceEditMesure.js]: move me in first mutation success

File: packages/app/src/components-v2/ServiceMesures/ServiceReactivateMesure.js [hunk 6]
```diff
@@ -52,11 +53,20 @@ export const ServiceReactivateMesure = props => {
...
                 variables: {
-                  antenne_id: mesure.antenne_id,
                   id: currentMesure,
-                  reason_extinction: values.reason_extinction
+                  reason_extinction: values.reason_extinction,
+                  service_id: mesure.service_id
                 }
               });
+              if (mesure.antenne_id) {
+                UpdateAntenneCounters({
```
Comment (kwiss) [inline packages/app/src/components-v2/ServiceMesures/ServiceReactivateMesure.js]: move me in first mutation success

**Pull request merged**